### ISSUE TITLE
Cant run rspec

### ISSUE THREAD

**Issue by mystycs** [learn-co-curriculum/orm-mapping-db-to-ruby-object-lab#13]
Cant run rspec and i usually have to run bundle install here, and it wont run, it says.  > Could not locate Gemfile

I am using nitrous.

**Comment by mystycs** [comment]
Maybe im wrong but running rspec does give me sqlite3 load errors.

**Comment by AnnJohn** [comment]
Hi @mystycs this might be a nitrous issue. Have you used the Ask a Question tool in Learn? Also you can file a bug report for this as well- if you hover over your profile pic in Learn, you'll see a menu option to File a Bug Report. Thanks.

**Comment by mystycs** [comment]
I reported it as a nitrous issue, they even went as far as reinstalling my nitrous container but it didnt help. 

I found that  i needed to run 
`gem install sqlite` and`gem install pry`

before running learn or rspec. Just letting you know because it took me a while to figure that out, and im not sure why it didnt work otherwise.

So i thought adding bundle install / gemfile to this would be beneficial for others.

Thanks!
